tests/lora/utils.py

```diff
@@ -1562,7 +1563,6 @@ def test_simple_inference_with_text_denoiser_multi_adapter_weighted(self):
                 "output with no lora and output with lora disabled should give same results",
             )
 
-    @skip_mps
```

Comment:
Why are we removing this?

Reply:
I added it earlier, assuming it was an MPS limitation. I was wrong, it was a legitimate failure.

Reply:
Then we should keep the `skip_mps` decorator right. Why the removal?

Reply:
It passes on MPS, I added it earlier in this PR based on an incorrect assumption that it was MPS causing the failure.

Reply:
Whoops, I wasn't the one to add it, but it does pass on MPS for AuraFlow at least.

Done in https://github.com/huggingface/diffusers/pull/10216/commits/fb5f5f7f24f4511a5ce76495149298f623fd6790.